scripts/hypothesistooling.py

```diff
@@ -82,6 +82,12 @@ def hash_for_name(name):
     ]).decode('ascii').strip()
 
 
+def is_ancestor(a, b):
+    return subprocess.call([
+        'git', '--is-ancestor', a, b
```

Comment:
I don't really understand why the old logic doesn't work either. But it's likely I've just misunderstood git's merge-base semantics.